### DISCUSSION TITLE
Incorrect icon for referenced draft PR

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [community/community#75294] [Mobile]
### Select Topic Area

Product Feedback

### Body

It's a small visual error.

In the timeline of a PR or question, the icon of a referenced draft PR is the same as that of a PR ready for review.

https://github.com/community/community/assets/40738104/e65c04de-b9b8-4273-a4fb-5842ace0db50

**Comment by HenSquared** [comment]
Great catch. I have opened up an issue with the Android team to take a look at this. Thank you for the feedback @Mysterious-Dev  :)

**Comment by mnemonicch** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It's a small visual error.
> 
> In the timeline of a PR or question, the icon of a referenced draft PR is the same as that of a PR ready for review.
> 
> 
> https://github.com/community/community/assets/40738104/e65c04de-b9b8-4273-a4fb-5842ace0db50
> 
> 

**__**

**Reply by Mysterious-Dev** [reply]
I do not understand what you mean :/
